### PR TITLE
fixed: multiline quotations are not always colored properly

### PR DESCRIPTION
**Pull request by vasily-kirichenko**

### CODE REVIEW

**Comment by vasily-kirichenko** [comment]
image

File: src/FSharpVSPowerTools.Core/SourceCodeClassifier.fs [hunk 1]
```diff
@@ -44,19 +44,17 @@ module private QuotationCategorizer =
...
-                            |> List.skipWhile ((<>) rquote)
-                            |> List.rev
-                        | _ -> 
-                            match tokens |> List.tryFind (fun t -> t.TokenName = "LQUOTE") with
-                            | Some lquote -> tokens |> List.skipWhile (fun t -> t <> lquote) 
-                            | _ -> tokens 
+                         if line = r.StartLine then
+                             tokens |> List.skipWhile (fun t -> t.LeftColumn < r.StartColumn)
+                         elif line = r.EndLine then
+                             tokens
```
Comment (dungpa) [inline src/FSharpVSPowerTools.Core/SourceCodeClassifier.fs]: Could we implement a `List.takeWhile` function at https://github.com/fsprojects/VisualFSharpPowerTools/blob/c92f530f0386bca4990265a0b7f8f64a65821edb/src/FSharpVSPowerTools.Core/Utils.fs#L18 and use it here?

Comment (vasily-kirichenko) [inline src/FSharpVSPowerTools.Core/SourceCodeClassifier.fs]: Sure. Will try to steal it from F# 4 Core.

**Comment by vasily-kirichenko** [comment]
Done.

**Comment by dungpa** [comment]
:+1:

**Pull request merged**